pennylane/ops/op_math/controlled.py

```diff
@@ -127,6 +127,11 @@ def cond_fn():
         available_eps = compiler.AvailableCompilers.names_entrypoints
         ops_loader = available_eps[active_jit]["ops"].load()
         return ops_loader.ctrl(op, control, control_values=control_values, work_wires=work_wires)
+    return _ctrl(op, control, control_values=control_values, work_wires=work_wires)
+
+
+def _ctrl(op, control, control_values=None, work_wires=None):
```

Comment:
I wonder if the default handling of control values could be left in the main function, put at the very beginning, so that catalyst won't have to have duplicate logic for that.

Reply:
Hmm, I see that Catalyst makes [additional checks](https://github.com/PennyLaneAI/catalyst/blob/178d042238828d3956f6c0f06a5e5a25a28a5513/frontend/catalyst/pennylane_extensions.py#L2283) before the standard conversions [here](https://github.com/PennyLaneAI/catalyst/blob/178d042238828d3956f6c0f06a5e5a25a28a5513/frontend/catalyst/pennylane_extensions.py#L1302). I would say it is better to let compilers/interpreters to handle the complete call..